src/components/NavigationBottom/NavigationBottom.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import NavbarData, { NavbarType } from '../_data/NavbarData/NavbarData';
+import './NavigationBottom.css';
+import { ArrowLeftCircle, ArrowRightCircle } from 'react-feather';
```

Comment:
You should not use this library. We have FontAwesome as a library installed already, so use that instead.